test/index.test.js

```diff
@@ -64,3 +64,13 @@ describe('diffPositionToFilePosition', () => {
     })
   })
 })
+
+describe('getDiff', () => {
+  it('returns accurate getDiff', async () => {
+    // Use this repo's git info, checking the 'package.json' file at a known commit
+    // https://github.com/pulsar-edit/whats-my-line/commit/71feedf0649bc342c9838d4f139043aa204dbead
+    const diff = await getDiff('./', 'package.json', '71feedf0649bc342c9838d4f139043aa204dbead');
```

Comment:
I think we might need two fixtures.

One that we commit (note down the SHA), then make some specific changes to (and commit again), _then_ generate a second fixture that's the exact diff of those changes since the initial commit of the first fixture.

Otherwise we won't ever be able to update `package.json`, which seems like a hindrance.

Reply:
Ahh this is a fantastic point! Thanks I didn't even consider how this would continually change, and how quickly our diff would change during the lifetime of this repo.

But I'll get right on that, creating a fixture file, along the lines of "Never Change" commit those, make a pointless edit, commit again, then we can use that in our tests.

Thanks for pointing this one out!

Reply:
Yeah, having some note(s) about "DO NOT CHANGE THIS FILE"  in fixture 1 is exactly along the lines of what I was thinking.